cli/testing/fake-editor.rs

```diff
@@ -50,6 +50,12 @@ fn main() {
                 let dest_path = edit_script_path.parent().unwrap().join(dest);
                 fs::copy(&args.file, dest_path).unwrap();
             }
+            ["dumppath", dest] => {
```

Comment:
nit: If we remove `expectpath`, I would rename this to `dump-path`.

Reply:
Done.